src/allmydata/test/web/test_web.py

```diff
@@ -1928,7 +1948,7 @@ def test_GET_FILEURL_badtype(self):
     def test_CSS_FILE(self):
         d = self.GET("/tahoe.css", followRedirect=True)
         def _check(res):
-            CSS_STYLE=re.compile('toolbar\s{.+text-align:\scenter.+toolbar-item.+display:\sinline',re.DOTALL)
+            CSS_STYLE=re.compile(b'toolbar\s{.+text-align:\scenter.+toolbar-item.+display:\sinline',re.DOTALL)
```

Comment:
:bird: :notes: :notes: :notes:

Reply:
I'm going to guess this is not actionable.